examples/huggingface-transformers/pipelines.py

```diff
@@ -527,42 +524,47 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
 
         answers = []
 
-        for i, sentence in enumerate(_inputs):
+        tokens = self.tokenizer(
+            _inputs,
+            return_tensors=self._framework,
+            truncation=TruncationStrategy.LONGEST_FIRST.value,
+            padding=PaddingStrategy.MAX_LENGTH.value,
+            return_special_tokens_mask=True,
+            return_offsets_mapping=self.tokenizer.is_fast,
+        )
 
-            tokens = self.tokenizer(
-                sentence,
-                return_tensors=self._framework,
-                truncation=TruncationStrategy.LONGEST_FIRST.value,
-                padding=PaddingStrategy.MAX_LENGTH.value,
-                return_special_tokens_mask=True,
-                return_offsets_mapping=self.tokenizer.is_fast,
-            )
-            if self.tokenizer.is_fast:
-                offset_mapping = tokens.pop("offset_mapping")[0]
-            elif offset_mappings:
-                offset_mapping = offset_mappings[i]
-            else:
-                offset_mapping = None
+        if self.tokenizer.is_fast:
+            offset_mapping = tokens.pop("offset_mapping")
+        elif not offset_mappings:
+            offset_mapping = [None] * len(_inputs)
+
+        special_tokens_mask = tokens.pop("special_tokens_mask")
 
-            special_tokens_mask = tokens.pop("special_tokens_mask")[0]
+        # Forward
+        _forward_pass = self._forward(tokens)
+        for entities_index, current_entities in enumerate(_forward_pass[0]):
```

Comment:
why `_forward_pass[0]`? what are in the other indices

Reply:
There are no other indices, the `_forward` function wraps it's output in a `list` with one element , so `_forward_pass[0]` just unwraps that.